### PR TITLE
build new module for support bundle

### PR DESCRIPTION
**Pull request by zbj6266**

### CODE REVIEW

File: cdap-master/src/main/java/io/cdap/cdap/common/MasterUtils.java
Comment (tivv) [inline cdap-master/src/main/java/io/cdap/cdap/common/MasterUtils.java]: Pass configuration set to 1

File: cdap-master/src/main/java/io/cdap/cdap/common/MasterUtils.java
Comment (tivv) [inline cdap-master/src/main/java/io/cdap/cdap/common/MasterUtils.java]: Double ";;"

File: cdap-support-bundle/pom.xml
Comment (tivv) [inline cdap-support-bundle/pom.xml]: I don't think we set it

File: cdap-support-bundle/pom.xml
Comment (tivv) [inline cdap-support-bundle/pom.xml]: Why do you need this one?

File: cdap-support-bundle/src/main/java/io/cdap/cdap/app/guice/SupportBundleRuntimeServiceModule.java
Comment (tivv) [inline cdap-support-bundle/src/main/java/io/cdap/cdap/app/guice/SupportBundleRuntimeServiceModule.java]: This won't work

File: cdap-support-bundle/src/main/java/io/cdap/cdap/app/guice/SupportBundleRuntimeServiceModule.java
Comment (tivv) [inline cdap-support-bundle/src/main/java/io/cdap/cdap/app/guice/SupportBundleRuntimeServiceModule.java]: This list looks strange. Usually distributed modules install same interfaces but different implementation

File: cdap-support-bundle/src/main/java/io/cdap/cdap/app/guice/SupportBundleRuntimeServiceModule.java
Comment (tivv) [inline cdap-support-bundle/src/main/java/io/cdap/cdap/app/guice/SupportBundleRuntimeServiceModule.java]: Why do you need artifact repository?

File: cdap-support-bundle/src/main/java/io/cdap/cdap/internal/app/services/SupportBundleInternalService.java
Comment (tivv) [inline cdap-support-bundle/src/main/java/io/cdap/cdap/internal/app/services/SupportBundleInternalService.java]: The 20000 looks big

File: cdap-standalone/src/main/java/io/cdap/cdap/StandaloneMain.java
Comment (tivv) [inline cdap-standalone/src/main/java/io/cdap/cdap/StandaloneMain.java]: nit: extra nl

**Review by tivv — COMMENTED** [review]
Seems fine for me, please get LGTM from @chtyim as I don't know this deep enough to approve

File: cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java
Comment (dli357) [inline cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java]: Please change "app-fabric" to support bundle

File: cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java
Comment (dli357) [inline cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java]: Are we running a separate instance of the SecureStore service in the support bundle pod?

File: cdap-support-bundle/pom.xml
Comment (dli357) [inline cdap-support-bundle/pom.xml:24]: Should this be 6.7.0-SNAPSHOT?

Comment (zbj6266) [reply cdap-support-bundle/pom.xml:24]: right now I think they haven't updated to 6.7 yet

File: cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java
Comment (zbj6266) [reply cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java]: yeah we don't need a separate instance for securestore service

File: cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java
Comment (dli357) [inline cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java]: We should probably be using `getDistributedModule()` here instead of `getMasterModule()`, otherwise we'll be running a standalone version of the authorization extension instead of calling back to app-fabric.

File: cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java
Comment (dli357) [inline cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java]: Do we need the AuthorizationModule? If not, please remove it, thanks!

File: cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java
Comment (dli357) [inline cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/SupportBundleServiceMain.java]: Do we need the SecureStoreServerModule? If not, please remove it, thanks!

**Review by dli357 — CHANGES_REQUESTED** [review]
LGTM from me pending resolution of most recent 3 comments, thanks!

**Review by dli357 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
